test/apps/iodemo/io_demo.cc

```diff
@@ -439,6 +446,12 @@ class P2pDemoCommon : public UcxContext {
             case UCS_MEMORY_TYPE_HOST:
                 buffer = UcxContext::memalign(ALIGNMENT, size,
                                               pool.name().c_str());
+                if (context != NULL && buffer != NULL) {
+                    if (context->map_buffer(size, buffer, &memh, 0) != UCS_OK) {
```

Comment:
maybe we should provide command line parameter to enable/disable buffers pre-registration so could also see that it improves performance

Reply:
done